Source/Diagnostics/FullDiagnostics.cpp

```diff
@@ -177,145 +177,138 @@ FullDiagnostics::InitializeFieldFunctorsRZopenPMD (int lev)
             warpx.get_pointer_current_fp(lev, dim)->nComp() == ncomp_multimodefab );
     }
 
-    amrex::Vector< std::string > temp_m_varnames(m_varnames);
```

Comment:
The variable `temp_m_varnames` is completely removed in this PR, as it is not needed anymore.